CMakeLists.txt

```diff
@@ -230,7 +230,7 @@ else()
     include(TestBigEndian)
     test_big_endian(NNG_BIG_ENDIAN)
     if (NNG_BIG_ENDIAN)
-        add_definitions(-DNNG_BIG_EDNDIAN=1)
+        add_definitions(-DNNG_BIG_ENDIAN=1)
```

Comment:
I fixed this about ten months ago upstream in commit f1cb2402734567b84e2b55129f98f9764167e13e

I also made more use of endian awareness at compile time to minimize byte switching,  The UDP transport uses little endian encoding for example, because as a new transport it felt kind of stupid to keep swapping bytes to satisfy architectures that nobody runs anymore.